sql: TestDropWhileBackfill failed

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1742350&tab=buildLog) on [master@261cd4e87d6d5a49e02d4192a3cfc18c1c5de1d0](https://github.com/cockroachdb/cockroach/commits/261cd4e87d6d5a49e02d4192a3cfc18c1c5de1d0):

```
W200210 22:41:31.666770 295496 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200210 22:41:31.667507 292397 ts/db.go:194  [n1,ts-poll] error writing time series data: result is ambiguous (server shutdown)
W200210 22:41:31.670129 295488 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 22:41:31.672115 295500 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 22:41:31.673658 295601 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
W200210 22:41:31.674071 295566 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: EOF:
W200210 22:41:31.677682 295478 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Unavailable desc = transport is closing:
W200210 22:41:31.677941 295637 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 22:41:31.678512 295544 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 22:41:31.678675 295556 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 22:41:31.679301 295537 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 22:41:31.680063 295618 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 22:41:31.680823 295517 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
W200210 22:41:31.704086 295512 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: EOF:
W200210 22:41:31.716201 292380 gossip/gossip.go:1518  [n1] no incoming or outgoing connections
I200210 22:41:31.761047 297047 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36505: context canceled
I200210 22:41:31.764679 297048 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:35235: context canceled
I200210 22:41:31.765819 296991 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:35235: context canceled
E200210 22:41:31.766284 296988 storage/queue.go:1035  [n1,consistencyChecker,s1,r17/1:/Table/2{1-2}] result is ambiguous (server shutdown)
I200210 22:41:31.767937 297050 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:42411: context canceled
I200210 22:41:31.768310 297128 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:36263: context canceled
I200210 22:41:31.768706 296990 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:39291: context canceled
I200210 22:41:31.771258 297049 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:36263: context canceled
E200210 22:41:31.770982 297053 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200210 22:41:31.770377 297127 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:39291: context canceled
I200210 22:41:31.774220 297130 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:42411: context canceled
I200210 22:41:31.771517 297129 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36505: context canceled
E200210 22:41:31.796939 297158 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200210 22:41:31.797326 297149 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:36263: context canceled
I200210 22:41:31.797560 297151 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:39291: context canceled
I200210 22:41:31.799185 297150 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:42411: context canceled
I200210 22:41:31.801233 297148 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:35235: context canceled
E200210 22:41:31.801356 297132 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200210 22:41:31.835264 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200210 22:41:31.835838 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200210 22:41:31.835963 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (split): node unavailable; try another peer
I200210 22:41:31.836162 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200210 22:41:31.836302 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200210 22:41:31.836411 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200210 22:41:31.836603 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200210 22:41:31.836735 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200210 22:41:31.836882 293681 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
I200210 22:41:31.882771 297054 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:42411: context canceled
I200210 22:41:31.886286 297056 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36505: context canceled
--- FAIL: TestDropWhileBackfill (11.82s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1742460&tab=buildLog) on [master@bf26e905f03a930b722b2b6e506a0f223ba6aef8](https://github.com/cockroachdb/cockroach/commits/bf26e905f03a930b722b2b6e506a0f223ba6aef8):

```
I200210 23:07:42.221683 297058 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:45967: context canceled
I200210 23:07:42.222206 297070 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:45967: context canceled
W200210 23:07:42.222456 292344 server/node.go:836  [n1,summaries] health alerts detected: {Alerts:[{StoreID:1 Category:METRICS Description:ranges.underreplicated Value:34}]}
I200210 23:07:42.223232 297071 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:34709: context canceled
I200210 23:07:42.223814 297072 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:35415: context canceled
E200210 23:07:42.224381 296962 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200210 23:07:42.223961 297059 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:44333: context canceled
E200210 23:07:42.225552 297040 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200210 23:07:42.224445 296960 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:37959: context canceled
I200210 23:07:42.224505 296958 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:34709: context canceled
I200210 23:07:42.226804 297057 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:37959: context canceled
W200210 23:07:42.229221 292344 server/node.go:807  [n1,summaries] error recording status summaries: node unavailable; try another peer
I200210 23:07:42.231036 297096 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:35415: context canceled
I200210 23:07:42.231430 297097 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:45967: context canceled
E200210 23:07:42.231213 296989 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200210 23:07:42.233000 297099 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:34709: context canceled
I200210 23:07:42.234338 296959 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:35415: context canceled
I200210 23:07:42.235002 297098 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:44333: context canceled
I200210 23:07:42.258865 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200210 23:07:42.259179 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200210 23:07:42.259280 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (split): node unavailable; try another peer
I200210 23:07:42.259578 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200210 23:07:42.259874 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200210 23:07:42.259970 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200210 23:07:42.260078 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200210 23:07:42.260168 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200210 23:07:42.260287 293809 storage/queue.go:524  [n4,s4] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
E200210 23:07:42.387278 296991 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200210 23:07:42.387751 297100 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200210 23:07:42.388440 296939 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200210 23:07:42.405925 297109 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200210 23:07:42.421787 297089 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
W200210 23:07:42.580724 292269 ts/db.go:194  [n1,ts-poll] error writing time series data: node unavailable; try another peer
E200210 23:07:42.620789 297154 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200210 23:07:42.638622 297117 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200210 23:07:42.647750 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200210 23:07:42.648141 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200210 23:07:42.648360 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (split): node unavailable; try another peer
I200210 23:07:42.648603 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200210 23:07:42.648768 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200210 23:07:42.648911 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200210 23:07:42.649077 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200210 23:07:42.649228 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200210 23:07:42.649467 293497 storage/queue.go:524  [n3,s3] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
--- FAIL: TestDropWhileBackfill (11.41s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1742548&tab=buildLog) on [master@d53e5f8a033cc352d10de80e5648d45389e710f7](https://github.com/cockroachdb/cockroach/commits/d53e5f8a033cc352d10de80e5648d45389e710f7):

```
I200210 23:48:13.969140 294097 sql/schema_changer.go:973  [n1,client=127.0.0.1:54078,user=root,scExec] schema change on test (53 v10) mutation 0 starting execution...
I200210 23:48:13.969912 294097 sql/schema_changer.go:1263  [n1,client=127.0.0.1:54078,user=root,scExec] waiting for a single version of table 53...
I200210 23:48:14.067025 294097 sql/schema_changer.go:1265  [n1,client=127.0.0.1:54078,user=root,scExec] waiting for a single version of table 53... done (at v 11)
I200210 23:48:14.096099 290358 util/stop/stopper.go:539  quiescing
I200210 23:48:14.097541 295341 util/stop/stopper.go:539  quiescing
I200210 23:48:14.099567 295337 util/stop/stopper.go:539  quiescing
I200210 23:48:14.104194 295338 util/stop/stopper.go:539  quiescing
W200210 23:48:14.105364 293685 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: EOF:
W200210 23:48:14.110116 292306 gossip/gossip.go:1518  [n5] no incoming or outgoing connections
I200210 23:48:14.111092 295339 util/stop/stopper.go:539  quiescing
I200210 23:48:14.116191 295340 util/stop/stopper.go:539  quiescing
W200210 23:48:14.121190 293462 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Unavailable desc = transport is closing:
W200210 23:48:14.122458 293484 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Unavailable desc = transport is closing:
W200210 23:48:14.123011 293622 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 23:48:14.127151 295286 storage/intentresolver/intent_resolver.go:820  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
W200210 23:48:14.128760 293445 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 23:48:14.131071 293538 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 23:48:14.134291 293589 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200210 23:48:14.141232 295315 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:35817: context canceled
W200210 23:48:14.143098 293422 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200210 23:48:14.144353 293512 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200210 23:48:14.143237 295314 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:42251: context canceled
W200210 23:48:14.145145 293576 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: EOF:
W200210 23:48:14.147064 293418 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
W200210 23:48:14.149423 293493 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200210 23:48:14.150700 293467 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200210 23:48:14.151275 293526 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: EOF:
W200210 23:48:14.151142 293469 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: EOF:
W200210 23:48:14.155665 293347 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
I200210 23:48:14.162674 290443 storage/queue.go:577  [n1,s1,r4/1:/System{/tsd-tse}] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200210 23:48:14.163387 290443 storage/queue.go:577  [n1,s1,r4/1:/System{/tsd-tse}] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200210 23:48:14.163597 295366 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:36477: context canceled
I200210 23:48:14.165776 295367 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:39531: context canceled
W200210 23:48:14.171588 290385 ts/db.go:194  [n1,ts-poll] error writing time series data: result is ambiguous (server shutdown)
W200210 23:48:14.198690 291591 gossip/gossip.go:1518  [n3] no incoming or outgoing connections
W200210 23:48:14.211548 291792 gossip/gossip.go:1518  [n4] no incoming or outgoing connections
W200210 23:48:14.216723 291245 gossip/gossip.go:1518  [n2] no incoming or outgoing connections
I200210 23:48:14.249695 290507 kv/transport_race.go:108  transport race promotion: ran 38 iterations on up to 1851 requests
I200210 23:48:14.293464 295271 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:39531: context canceled
I200210 23:48:14.293542 295272 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:35817: context canceled
I200210 23:48:14.294111 295274 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:42251: context canceled
E200210 23:48:14.294270 295229 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200210 23:48:14.294632 295275 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200210 23:48:14.294856 295273 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:39673: context canceled
--- FAIL: TestDropWhileBackfill (11.88s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1742621&tab=buildLog) on [master@1551d49cc9abcc791bd7295b01d5a16a5fd93292](https://github.com/cockroachdb/cockroach/commits/1551d49cc9abcc791bd7295b01d5a16a5fd93292):

```
I200211 00:24:11.301563 292307 sql/lease.go:327  [n1,client=127.0.0.1:60958,user=root,scExec] waiting for 1 leases to expire: desc=[{test 53 8}]
I200211 00:24:11.316942 294233 sql/lease.go:924  new lease: 53("test") ver=9:1581380964.450830335,0, refcount=0
I200211 00:24:11.419985 292307 sql/schema_changer.go:1265  [n1,client=127.0.0.1:60958,user=root,scExec] waiting for a single version of table 53... done (at v 9)
W200211 00:24:11.439662 292307 sql/exec_util.go:1152  [n1,client=127.0.0.1:60958,user=root] error executing schema change: context deadline exceeded
I200211 00:24:11.494077 293862 sql/sqlbase/structured.go:1497  [n1,client=127.0.0.1:60966,user=root] publish: descID=53 (test) version=10 mtime=1970-01-01 00:00:00 +0000 UTC
I200211 00:24:11.650872 293862 sql/sqlbase/structured.go:1497  [n1,client=127.0.0.1:60966,user=root] publish: descID=53 (test) version=10 mtime=1970-01-01 00:00:00 +0000 UTC
I200211 00:24:11.706151 293862 sql/event_log.go:132  [n1,client=127.0.0.1:60966,user=root] Event: "drop_table", target: 53, info: {TableName:t.public.test Statement:DROP TABLE t.public.test User:root CascadeDroppedViews:[]}
I200211 00:24:11.765870 293862 sql/lease.go:327  [n1,client=127.0.0.1:60966,user=root,scExec] waiting for 1 leases to expire: desc=[{test 53 9}]
I200211 00:24:11.827514 293862 sql/sqlbase/structured.go:1497  [n1,client=127.0.0.1:60966,user=root,scExec] publish: descID=53 (test) version=11 mtime=1970-01-01 00:00:00 +0000 UTC
I200211 00:24:11.969888 293862 sql/schema_changer.go:973  [n1,client=127.0.0.1:60966,user=root,scExec] schema change on test (53 v10) mutation 0 starting execution...
I200211 00:24:11.970104 293862 sql/schema_changer.go:1263  [n1,client=127.0.0.1:60966,user=root,scExec] waiting for a single version of table 53...
I200211 00:24:12.012795 293862 sql/schema_changer.go:1265  [n1,client=127.0.0.1:60966,user=root,scExec] waiting for a single version of table 53... done (at v 11)
I200211 00:24:12.068605 290054 util/stop/stopper.go:539  quiescing
I200211 00:24:12.069214 294727 util/stop/stopper.go:539  quiescing
I200211 00:24:12.069397 294728 util/stop/stopper.go:539  quiescing
I200211 00:24:12.069586 294731 util/stop/stopper.go:539  quiescing
I200211 00:24:12.091709 290068 kv/transport_race.go:108  transport race promotion: ran 29 iterations on up to 1719 requests
I200211 00:24:12.069463 294729 util/stop/stopper.go:539  quiescing
W200211 00:24:12.094185 293206 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200211 00:24:12.096599 294539 storage/intentresolver/intent_resolver.go:820  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
W200211 00:24:12.099511 293315 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.101905 293140 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Unavailable desc = transport is closing:
W200211 00:24:12.103612 293314 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.118728 293210 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.125235 293310 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.126694 293234 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.134713 292068 gossip/gossip.go:1518  [n5] no incoming or outgoing connections
W200211 00:24:12.136928 293237 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.140812 293177 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.142667 293288 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: EOF:
I200211 00:24:12.069546 294730 util/stop/stopper.go:539  quiescing
W200211 00:24:12.143503 293183 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200211 00:24:12.144323 291659 gossip/gossip.go:1518  [n4] no incoming or outgoing connections
W200211 00:24:12.144867 293335 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.145881 293336 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 00:24:12.146283 293238 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: EOF:
W200211 00:24:12.151672 291086 gossip/gossip.go:1518  [n2] no incoming or outgoing connections
W200211 00:24:12.153950 291324 gossip/gossip.go:1518  [n3] no incoming or outgoing connections
W200211 00:24:12.155028 293287 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: EOF:
E200211 00:24:12.252839 294724 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 00:24:12.252915 294743 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41599: context canceled
I200211 00:24:12.253457 294744 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:34155: context canceled
I200211 00:24:12.254666 294745 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:34547: context canceled
I200211 00:24:12.255033 294746 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:34347: context canceled
--- FAIL: TestDropWhileBackfill (9.35s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1742713&tab=buildLog) on [master@d83713fb7d9d52e45e488bf5c56e22cf1041fdc6](https://github.com/cockroachdb/cockroach/commits/d83713fb7d9d52e45e488bf5c56e22cf1041fdc6):

```
I200211 02:34:07.076217 297377 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:46793: context canceled
W200211 02:34:07.084809 295678 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 02:34:07.109789 295700 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 02:34:07.110184 297342 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:46793: context canceled
E200211 02:34:07.111078 297430 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
W200211 02:34:07.113215 295922 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: EOF:
I200211 02:34:07.114908 297343 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:33881: context canceled
I200211 02:34:07.116045 297341 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:38029: context canceled
W200211 02:34:07.118141 296005 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 02:34:07.118196 293749 gossip/gossip.go:1518  [n2] no incoming or outgoing connections
W200211 02:34:07.120867 295835 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 02:34:07.123938 294357 gossip/gossip.go:1518  [n4] no incoming or outgoing connections
W200211 02:34:07.124406 295793 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
I200211 02:34:07.124910 297435 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:38029: context canceled
I200211 02:34:07.127130 297432 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:46793: context canceled
I200211 02:34:07.127301 297434 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:33667: context canceled
I200211 02:34:07.128901 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200211 02:34:07.129009 297433 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:33881: context canceled
I200211 02:34:07.134774 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200211 02:34:07.135303 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (split): node unavailable; try another peer
I200211 02:34:07.135583 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 02:34:07.135843 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 02:34:07.136124 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 02:34:07.136340 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 02:34:07.136639 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 02:34:07.136878 293665 storage/queue.go:577  [n2,s2] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
I200211 02:34:07.176697 297464 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:33881: context canceled
I200211 02:34:07.178409 297461 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36029: context canceled
I200211 02:34:07.180002 297462 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:38029: context canceled
I200211 02:34:07.181506 297463 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:33667: context canceled
I200211 02:34:07.186168 297469 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36029: context canceled
I200211 02:34:07.187062 297471 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:33881: context canceled
I200211 02:34:07.188368 297470 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:46793: context canceled
I200211 02:34:07.189283 297472 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:33667: context canceled
E200211 02:34:07.189295 297474 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 02:34:07.211816 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200211 02:34:07.212336 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200211 02:34:07.212607 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (split): node unavailable; try another peer
I200211 02:34:07.212695 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 02:34:07.212887 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 02:34:07.213148 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 02:34:07.213309 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 02:34:07.213386 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 02:34:07.213463 294509 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
--- FAIL: TestDropWhileBackfill (9.74s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1742760&tab=buildLog) on [master@d83713fb7d9d52e45e488bf5c56e22cf1041fdc6](https://github.com/cockroachdb/cockroach/commits/d83713fb7d9d52e45e488bf5c56e22cf1041fdc6):

```
W200211 04:27:38.558558 292138 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: EOF:
I200211 04:27:38.568559 293599 util/stop/stopper.go:539  quiescing
W200211 04:27:38.551267 292231 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 04:27:38.577662 293678 storage/intentresolver/intent_resolver.go:820  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
W200211 04:27:38.580256 292002 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: EOF:
W200211 04:27:38.580979 292079 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200211 04:27:38.585368 291775 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: EOF:
W200211 04:27:38.588861 289942 gossip/gossip.go:1518  [n2] no incoming or outgoing connections
W200211 04:27:38.589203 292010 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
W200211 04:27:38.591505 290582 gossip/gossip.go:1518  [n4] no incoming or outgoing connections
W200211 04:27:38.592273 291855 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 04:27:38.595752 293848 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36385: context canceled
I200211 04:27:38.595991 293847 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:33129: context canceled
I200211 04:27:38.597699 293845 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:44779: context canceled
I200211 04:27:38.597929 293846 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41125: context canceled
W200211 04:27:38.599388 291997 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 04:27:38.604290 292181 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 04:27:38.605367 292202 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 04:27:38.607108 291860 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Unavailable desc = transport is closing:
W200211 04:27:38.607594 292014 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 04:27:38.611236 293597 util/stop/stopper.go:539  quiescing
I200211 04:27:38.616518 289152 kv/transport_race.go:108  transport race promotion: ran 68 iterations on up to 1843 requests
W200211 04:27:38.592669 291978 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
W200211 04:27:38.623088 292036 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 04:27:38.577931 290896 gossip/gossip.go:1518  [n5] no incoming or outgoing connections
W200211 04:27:38.613420 290233 gossip/gossip.go:1518  [n3] no incoming or outgoing connections
W200211 04:27:38.639833 292156 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Unavailable desc = transport is closing:
I200211 04:27:38.611430 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200211 04:27:38.642201 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200211 04:27:38.642477 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (split): node unavailable; try another peer
I200211 04:27:38.642758 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 04:27:38.642982 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 04:27:38.643160 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 04:27:38.643373 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 04:27:38.643505 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 04:27:38.643710 290734 storage/queue.go:577  [n4,s4] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
I200211 04:27:38.681307 293841 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:33129: context canceled
I200211 04:27:38.681678 293839 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:40983: context canceled
I200211 04:27:38.682666 293840 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41125: context canceled
I200211 04:27:38.683699 293838 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36385: context canceled
I200211 04:27:38.721394 293738 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:40983: context canceled
I200211 04:27:38.721642 293740 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36385: context canceled
I200211 04:27:38.721941 293739 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:44779: context canceled
I200211 04:27:38.724809 293741 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:33129: context canceled
--- FAIL: TestDropWhileBackfill (8.73s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1742811&tab=buildLog) on [master@f23d6b66e0516125e107d2970357392ede35c46d](https://github.com/cockroachdb/cockroach/commits/f23d6b66e0516125e107d2970357392ede35c46d):

```
I200211 05:29:32.039222 293676 sql/sqlbase/structured.go:1497  [n1,client=127.0.0.1:43736,user=root] publish: descID=53 (test) version=10 mtime=1970-01-01 00:00:00 +0000 UTC
I200211 05:29:32.094612 293676 sql/event_log.go:132  [n1,client=127.0.0.1:43736,user=root] Event: "drop_table", target: 53, info: {TableName:t.public.test Statement:DROP TABLE t.public.test User:root CascadeDroppedViews:[]}
I200211 05:29:32.147359 293676 sql/lease.go:327  [n1,client=127.0.0.1:43736,user=root,scExec] waiting for 1 leases to expire: desc=[{test 53 9}]
I200211 05:29:32.222524 293676 sql/sqlbase/structured.go:1497  [n1,client=127.0.0.1:43736,user=root,scExec] publish: descID=53 (test) version=11 mtime=1970-01-01 00:00:00 +0000 UTC
I200211 05:29:32.304956 293676 sql/schema_changer.go:973  [n1,client=127.0.0.1:43736,user=root,scExec] schema change on test (53 v10) mutation 0 starting execution...
I200211 05:29:32.305501 293676 sql/schema_changer.go:1263  [n1,client=127.0.0.1:43736,user=root,scExec] waiting for a single version of table 53...
I200211 05:29:32.337519 293676 sql/schema_changer.go:1265  [n1,client=127.0.0.1:43736,user=root,scExec] waiting for a single version of table 53... done (at v 11)
I200211 05:29:32.354293 290003 util/stop/stopper.go:539  quiescing
I200211 05:29:32.354785 295101 util/stop/stopper.go:539  quiescing
I200211 05:29:32.355180 295102 util/stop/stopper.go:539  quiescing
I200211 05:29:32.355345 295103 util/stop/stopper.go:539  quiescing
I200211 05:29:32.355596 295104 util/stop/stopper.go:539  quiescing
I200211 05:29:32.356188 295105 util/stop/stopper.go:539  quiescing
W200211 05:29:32.356712 295073 storage/intentresolver/intent_resolver.go:820  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
I200211 05:29:32.356893 290198 kv/transport_race.go:108  transport race promotion: ran 39 iterations on up to 2075 requests
W200211 05:29:32.365049 291569 gossip/gossip.go:1518  [n4] no incoming or outgoing connections
W200211 05:29:32.365781 290893 gossip/gossip.go:1518  [n2] no incoming or outgoing connections
W200211 05:29:32.370188 292995 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200211 05:29:32.373442 293206 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: rpc error: code = Unavailable desc = transport is closing:
W200211 05:29:32.377087 293063 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
W200211 05:29:32.388886 293095 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 05:29:32.394084 293037 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 05:29:32.395864 295016 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:37015: context canceled
I200211 05:29:32.398217 295017 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41917: context canceled
I200211 05:29:32.401929 294911 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41917: node unavailable; try another peer
W200211 05:29:32.404132 293096 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Unavailable desc = transport is closing:
W200211 05:29:32.405042 293008 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 05:29:32.408103 294912 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:44167: node unavailable; try another peer
W200211 05:29:32.411317 293118 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 05:29:32.418174 293179 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 05:29:32.419006 293107 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 05:29:32.419739 292801 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200211 05:29:32.422222 293092 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 05:29:32.424958 293174 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 05:29:32.427530 295084 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:33201: context canceled
W200211 05:29:32.438514 292986 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 05:29:32.442082 291905 gossip/gossip.go:1518  [n5] no incoming or outgoing connections
W200211 05:29:32.449639 293143 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 05:29:32.454946 295141 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:44167: context canceled
I200211 05:29:32.455183 294915 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:33201: context canceled
I200211 05:29:32.455384 295142 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:35055: context canceled
E200211 05:29:32.455760 295130 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200211 05:29:32.456445 295106 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 05:29:32.456739 294916 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41917: context canceled
--- FAIL: TestDropWhileBackfill (9.17s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1742849&tab=buildLog) on [master@f23d6b66e0516125e107d2970357392ede35c46d](https://github.com/cockroachdb/cockroach/commits/f23d6b66e0516125e107d2970357392ede35c46d):

```
W200211 06:56:56.863135 290015 storage/store.go:1538  [n1,s1,r1/1:/{Min-System/NodeL…}] could not gossip first range descriptor: node unavailable; try another peer
W200211 06:56:56.865675 293116 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: EOF:
I200211 06:56:56.866566 294388 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:44799: context canceled
W200211 06:56:56.868786 293016 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: EOF:
I200211 06:56:56.868906 294533 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:43113: context canceled
W200211 06:56:56.871608 293778 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: EOF:
W200211 06:56:56.871660 292891 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: EOF:
W200211 06:56:56.873103 293061 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: EOF:
I200211 06:56:56.875488 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200211 06:56:56.875930 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200211 06:56:56.876083 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (split): node unavailable; try another peer
I200211 06:56:56.876343 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 06:56:56.876538 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 06:56:56.876685 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 06:56:56.876774 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 06:56:56.877027 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 06:56:56.877137 290045 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
W200211 06:56:56.878781 293028 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: EOF:
W200211 06:56:56.884891 292927 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: EOF:
W200211 06:56:56.904019 292984 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 06:56:56.910096 294538 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:45449: context canceled
I200211 06:56:56.911189 294540 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:43113: context canceled
I200211 06:56:56.911782 294541 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:46123: context canceled
E200211 06:56:56.912965 294488 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 06:56:56.913013 294539 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:44799: context canceled
I200211 06:56:56.919008 289952 kv/transport_race.go:108  transport race promotion: ran 26 iterations on up to 1689 requests
I200211 06:56:56.944294 294521 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:46123: context canceled
I200211 06:56:56.945935 294518 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:46341: context canceled
I200211 06:56:56.947567 294520 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:44799: context canceled
I200211 06:56:56.950902 294519 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:45449: context canceled
E200211 06:56:56.968560 294304 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 06:56:56.969246 294489 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:46341: context canceled
I200211 06:56:56.969667 294491 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:43113: context canceled
I200211 06:56:56.971193 294492 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:45449: context canceled
I200211 06:56:56.973629 294490 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:46123: context canceled
I200211 06:56:56.975980 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200211 06:56:56.976836 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200211 06:56:56.977831 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (split): node unavailable; try another peer
I200211 06:56:56.978445 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 06:56:56.978926 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 06:56:56.979225 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 06:56:56.979640 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 06:56:56.979977 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 06:56:56.980201 292001 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
--- FAIL: TestDropWhileBackfill (9.49s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1743996&tab=buildLog) on [master@f23d6b66e0516125e107d2970357392ede35c46d](https://github.com/cockroachdb/cockroach/commits/f23d6b66e0516125e107d2970357392ede35c46d):

```
I200211 07:17:39.940069 291441 kv/transport_race.go:108  transport race promotion: ran 30 iterations on up to 2058 requests
W200211 07:17:39.942874 294629 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Unavailable desc = transport is closing:
W200211 07:17:39.944851 294528 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: rpc error: code = Unavailable desc = transport is closing:
W200211 07:17:39.945581 292405 gossip/gossip.go:1518  [n2] no incoming or outgoing connections
W200211 07:17:39.948997 293017 gossip/gossip.go:1518  [n4] no incoming or outgoing connections
W200211 07:17:39.951419 294466 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: EOF:
W200211 07:17:39.957395 294468 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:39.963319 294575 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:39.965249 294520 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:39.978557 294475 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:39.978626 294480 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: EOF:
W200211 07:17:39.978969 294372 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:39.987634 294655 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:39.995644 294599 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:39.996883 294622 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:39.999168 294581 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 07:17:40.000964 294473 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
I200211 07:17:40.005450 296463 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:34293: context canceled
I200211 07:17:40.007767 296461 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:40651: context canceled
I200211 07:17:40.008487 296462 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:45153: context canceled
W200211 07:17:40.010156 294545 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 07:17:40.011037 296479 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:34293: context canceled
I200211 07:17:40.013916 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200211 07:17:40.014340 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200211 07:17:40.014660 296476 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:33403: context canceled
I200211 07:17:40.014999 296477 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36069: context canceled
I200211 07:17:40.015066 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (split): node unavailable; try another peer
I200211 07:17:40.022935 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 07:17:40.023157 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 07:17:40.023347 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 07:17:40.023537 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 07:17:40.023680 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 07:17:40.023807 293561 storage/queue.go:577  [n5,s5] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
I200211 07:17:40.025324 296478 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:45153: context canceled
W200211 07:17:40.026472 293314 gossip/gossip.go:1518  [n5] no incoming or outgoing connections
I200211 07:17:40.034311 296538 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:40651: context canceled
I200211 07:17:40.036106 296541 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:33403: context canceled
I200211 07:17:40.036555 296304 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:34293: context canceled
I200211 07:17:40.037845 296540 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:34293: context canceled
W200211 07:17:40.038002 291579 storage/store.go:1538  [n1,s1,r1/1:/{Min-System/NodeL…}] could not gossip first range descriptor: node unavailable; try another peer
I200211 07:17:40.039803 296539 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:45153: context canceled
I200211 07:17:40.040292 296305 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36069: context canceled
I200211 07:17:40.042833 296306 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:33403: context canceled
I200211 07:17:40.043738 296307 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:40651: context canceled
--- FAIL: TestDropWhileBackfill (9.32s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1744281&tab=buildLog) on [master@f23d6b66e0516125e107d2970357392ede35c46d](https://github.com/cockroachdb/cockroach/commits/f23d6b66e0516125e107d2970357392ede35c46d):

```
I200211 09:09:15.812523 293119 sql/schema_changer.go:1265  [n1,client=127.0.0.1:45186,user=root,scExec] waiting for a single version of table 53... done (at v 11)
I200211 09:09:15.840891 289517 util/stop/stopper.go:539  quiescing
I200211 09:09:15.841477 294256 util/stop/stopper.go:539  quiescing
I200211 09:09:15.841827 294258 util/stop/stopper.go:539  quiescing
I200211 09:09:15.842025 294259 util/stop/stopper.go:539  quiescing
I200211 09:09:15.841606 294257 util/stop/stopper.go:539  quiescing
W200211 09:09:15.859282 292554 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: rpc error: code = Unavailable desc = transport is closing:
W200211 09:09:15.860342 292498 storage/raft_transport.go:637  [n4] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 09:09:15.862943 292525 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: EOF:
I200211 09:09:15.843197 294255 util/stop/stopper.go:539  quiescing
W200211 09:09:15.879361 292602 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 4: EOF:
W200211 09:09:15.882767 292531 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 09:09:15.886578 292647 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 09:09:15.887015 292676 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: EOF:
W200211 09:09:15.888944 290455 gossip/gossip.go:1518  [n2] no incoming or outgoing connections
W200211 09:09:15.892097 291493 gossip/gossip.go:1518  [n5] no incoming or outgoing connections
W200211 09:09:15.895976 292586 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: EOF:
W200211 09:09:15.899795 292569 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Unavailable desc = transport is closing:
W200211 09:09:15.904758 292701 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
I200211 09:09:15.917402 294313 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:42389: node unavailable; try another peer
W200211 09:09:15.918626 292741 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 09:09:15.919895 294312 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:37509: node unavailable; try another peer
W200211 09:09:15.924664 294089 storage/intentresolver/intent_resolver.go:820  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
W200211 09:09:15.927739 292547 storage/raft_transport.go:637  [n5] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 09:09:15.931490 292418 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 5: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 09:09:15.936131 292655 storage/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: rpc error: code = Unavailable desc = transport is closing:
W200211 09:09:15.938073 292652 storage/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200211 09:09:15.939891 290757 gossip/gossip.go:1518  [n3] no incoming or outgoing connections
I200211 09:09:15.994119 294090 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:37509: context canceled
I200211 09:09:15.994825 294092 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:35039: context canceled
E200211 09:09:15.995389 294094 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 09:09:15.994330 294091 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36171: context canceled
I200211 09:09:15.995764 294093 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:45269: context canceled
I200211 09:09:15.997935 294147 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36171: context canceled
I200211 09:09:15.998810 294342 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:45269: context canceled
E200211 09:09:15.999275 294209 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 09:09:15.999478 294148 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:42389: context canceled
I200211 09:09:15.999960 294341 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:35039: context canceled
W200211 09:09:16.001576 289500 gossip/gossip.go:1518  [n1] no incoming or outgoing connections
I200211 09:09:16.005096 289491 kv/transport_race.go:108  transport race promotion: ran 104 iterations on up to 1842 requests
I200211 09:09:16.016168 294358 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:45269: context canceled
I200211 09:09:16.016264 294359 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:42389: context canceled
I200211 09:09:16.017661 294357 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:36171: context canceled
I200211 09:09:16.018824 294360 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:37509: context canceled
--- FAIL: TestDropWhileBackfill (8.27s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1744323&tab=buildLog) on [master@cdf805f63e2c448a99086fdea500ef627015fabc](https://github.com/cockroachdb/cockroach/commits/cdf805f63e2c448a99086fdea500ef627015fabc):

```
I200211 10:36:30.344870 292836 storage/queue.go:577  [n3,s3] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 10:36:30.345026 292836 storage/queue.go:577  [n3,s3] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 10:36:30.345126 292836 storage/queue.go:577  [n3,s3] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 10:36:30.345265 292836 storage/queue.go:577  [n3,s3] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 10:36:30.345562 292836 storage/queue.go:577  [n3,s3] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 10:36:30.345858 292836 storage/queue.go:577  [n3,s3] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
W200211 10:36:30.358436 294572 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
I200211 10:36:30.388274 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200211 10:36:30.388668 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200211 10:36:30.388827 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (split): node unavailable; try another peer
I200211 10:36:30.389027 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 10:36:30.389199 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 10:36:30.389452 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 10:36:30.389527 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 10:36:30.389722 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 10:36:30.389994 291679 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
I200211 10:36:30.400126 296759 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:40193: context canceled
I200211 10:36:30.400728 296761 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:33893: context canceled
I200211 10:36:30.402536 296760 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:44437: context canceled
I200211 10:36:30.407637 296752 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:33893: context canceled
I200211 10:36:30.403177 296751 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:44437: context canceled
I200211 10:36:30.405128 296750 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:40193: context canceled
I200211 10:36:30.406516 296749 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:37045: context canceled
I200211 10:36:30.409873 296762 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:34413: context canceled
E200211 10:36:30.410034 296765 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
W200211 10:36:30.451720 291651 storage/store.go:1538  [n1,s1,r1/1:/{Min-System/NodeL…}] could not gossip first range descriptor: node unavailable; try another peer
E200211 10:36:30.461428 296609 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 10:36:30.461604 296735 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:40193: context canceled
I200211 10:36:30.462249 296738 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:44437: context canceled
I200211 10:36:30.463309 296737 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:34413: context canceled
E200211 10:36:30.463336 296670 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 10:36:30.493237 296767 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:33893: context canceled
I200211 10:36:30.494267 296769 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:37045: context canceled
I200211 10:36:30.494730 296768 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:34413: context canceled
E200211 10:36:30.494963 296687 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 10:36:30.495146 296770 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:44437: context canceled
E200211 10:36:30.495343 296671 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
W200211 10:36:30.508717 291651 storage/store.go:1538  [n1,s1,r1/1:/{Min-System/NodeL…}] could not gossip first range descriptor: node unavailable; try another peer
I200211 10:36:30.540216 296673 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:34413: context canceled
I200211 10:36:30.541441 296674 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:40193: context canceled
E200211 10:36:30.541730 296725 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 10:36:30.542013 296675 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:37045: context canceled
I200211 10:36:30.557250 296672 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:33893: context canceled
W200211 10:36:30.564186 291505 ts/db.go:194  [n1,ts-poll] error writing time series data: result is ambiguous (server shutdown)
--- FAIL: TestDropWhileBackfill (11.75s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql).TestDropWhileBackfill failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1744495&tab=buildLog) on [master@7b28515785549a2c8b94be9f82f6ff5e28db8a64](https://github.com/cockroachdb/cockroach/commits/7b28515785549a2c8b94be9f82f6ff5e28db8a64):

```
I200211 16:10:13.416588 299209 util/stop/stopper.go:539  quiescing
W200211 16:10:13.421376 294972 gossip/gossip.go:1518  [n3] no incoming or outgoing connections
W200211 16:10:13.421403 296499 storage/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
W200211 16:10:13.422365 295367 gossip/gossip.go:1518  [n4] no incoming or outgoing connections
I200211 16:10:13.422679 299210 util/stop/stopper.go:539  quiescing
I200211 16:10:13.422894 299208 util/stop/stopper.go:539  quiescing
I200211 16:10:13.453584 299160 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:46145: context canceled
I200211 16:10:13.456133 299161 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:44155: context canceled
I200211 16:10:13.458289 299162 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:42205: context canceled
I200211 16:10:13.460199 299163 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:42717: context canceled
I200211 16:10:13.466088 299171 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41989: context canceled
I200211 16:10:13.467875 299168 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:42205: context canceled
I200211 16:10:13.471051 299169 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:44155: context canceled
I200211 16:10:13.473987 299170 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:42717: context canceled
W200211 16:10:13.487559 295424 ts/db.go:194  [n4,ts-poll] error writing time series data: failed to send RPC: sending to all 1 replicas failed; last error: <nil> rpc error: code = Unavailable desc = transport is closing
I200211 16:10:13.526820 299040 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:46145: context canceled
I200211 16:10:13.527985 299041 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:44155: context canceled
I200211 16:10:13.528704 299039 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41989: context canceled
E200211 16:10:13.530384 299043 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 16:10:13.531013 299042 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:42205: context canceled
I200211 16:10:13.573555 293871 storage/queue.go:577  [n1,s1,r4/1:/System{/tsd-tse}] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200211 16:10:13.579297 299254 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:42717: context canceled
I200211 16:10:13.579374 299253 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:42205: context canceled
I200211 16:10:13.580314 299044 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:46145: context canceled
E200211 16:10:13.584031 299256 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 16:10:13.584384 299255 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41989: context canceled
I200211 16:10:13.594791 293911 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200211 16:10:13.595272 293911 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200211 16:10:13.595503 293911 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (split): node unavailable; try another peer
I200211 16:10:13.595592 293911 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200211 16:10:13.595722 293911 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200211 16:10:13.596083 293911 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200211 16:10:13.596229 293911 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200211 16:10:13.596450 293911 storage/queue.go:577  [n1,s1] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
I200211 16:10:13.617316 299257 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n5: failed to connect to n5 at 127.0.0.1:41989: context canceled
I200211 16:10:13.619992 299259 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:44155: context canceled
I200211 16:10:13.619705 299258 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:46145: context canceled
I200211 16:10:13.617469 299260 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n4: failed to connect to n4 at 127.0.0.1:42717: context canceled
E200211 16:10:13.620111 299234 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200211 16:10:13.650654 299148 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
E200211 16:10:13.652755 299213 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
I200211 16:10:13.660104 293941 kv/transport_race.go:108  transport race promotion: ran 102 iterations on up to 2054 requests
W200211 16:10:13.708345 295926 ts/db.go:197  [n5,ts-poll] node unavailable; try another peer
E200211 16:10:13.739016 299156 vendor/google.golang.org/grpc/pickfirst.go:61  pickfirstBalancer: failed to NewSubConn: rpc error: code = Canceled desc = grpc: the client connection is closing
--- FAIL: TestDropWhileBackfill (15.80s)
    schema_changer_test.go:840: pq: context deadline exceeded


Failed to find issue assignee: 
old friend vivekmenezes is friendless
```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDropWhileBackfill PKG=./pkg/sql TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDropWhileBackfill.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/sql/schema_changer_test.go
+++ b/pkg/sql/schema_changer_test.go
@@ -611,7 +611,6 @@ CREATE UNIQUE INDEX vidx ON t.test (v);
 // successfully complete without deleting the data.
 func TestDropWhileBackfill(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	t.Skip("https://github.com/cockroachdb/cockroach/issues/44944")
 	// protects backfillNotification
 	var mu syncutil.Mutex
 	backfillNotification := make(chan struct{})
@@ -639,8 +638,6 @@ func TestDropWhileBackfill(t *testing.T) {
 		SQLSchemaChanger: &sql.SchemaChangerTestingKnobs{
 			BackfillChunkSize: chunkSize,
 		},
-		// Disable GC job.
-		GCJob: &sql.GCJobTestingKnobs{RunBeforeResume: func(_ int64) error { select {} }},
 		DistSQL: &execinfra.TestingKnobs{
 			RunBeforeBackfillChunk: func(sp roachpb.Span) error {
 				if partialBackfillDone.Load().(bool) {

